domain/src/main/java/com/anwera97/domain/mappers/CompanionMapper.kt

```diff
@@ -39,7 +39,7 @@ private fun toHashMap(results: List<ResultModel>): Map<String, ResultModel> {
     return map
 }
 
-private fun PayerWithExpendituresAndDebtors.getTotalPayment(): Double {
+private fun com.anwera97.data.composedclasses.PayerWithExpendituresAndDebtors.getTotalPayment(): Double {
```

Comment:
you can add this as a suffix? to make it shorter

Reply:
Sure